x-pack/tasks/build.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { resolve } from 'path';
+import { writeFileSync } from 'fs';
+
+// @ts-ignore
```

Comment:
nit: I like to comment after an ignore to indicate why... plus it makes it easier to grep how many ignores we have and why... e.g:

`// @ts-ignore Untyped local`
`// @ts-ignore Untyped dependency`
`// @ts-ignore I'm casting this to all because I don't know any better`